components/places/examples/places_utils.rs

```diff
@@ -0,0 +1,232 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use places::storage::bookmarks::{
+    fetch_tree, insert_tree, BookmarkNode, BookmarkRootGuid, BookmarkTreeNode, FolderNode,
+    SeparatorNode,
+};
+use places::types::{BookmarkType, SyncGuid, Timestamp};
+use places::PlacesDb;
+
+use serde_derive::*;
+use sql_support::ConnExt;
+use std::fs::File;
+use std::io::{BufReader, BufWriter};
+use url::Url;
+
+type Result<T> = std::result::Result<T, failure::Error>;
+
+fn init_logging() {
+    // Explicitly ignore some rather noisy crates. Turn on trace for everyone else.
+    let spec = "trace,tokio_threadpool=warn,tokio_reactor=warn,tokio_core=warn,tokio=warn,hyper=warn,want=warn,mio=warn,reqwest=warn";
```

Comment:
God we need to put this somewhere shared. Some support/examples or something. (it's fine for now, just a thought).